src/main/java/me/totalfreedom/totalfreedommod/command/Command_linkreddit.java

```diff
@@ -38,7 +38,7 @@ public boolean run(final CommandSender sender, final Player playerSender, final
 
             try
             {
-                plugin.rd.sendModMessage(username, "Link Code", "Please run the following in-game to link your Reddit account: /linkreddit code " + code);
+                plugin.rd.sendModMessage(username, "Link Code", "Please run the following in-game to link your Reddit account: /linkreddit code " + code + ".");
```

Comment:
Think this was my point before, if we're giving instructions or codes we should probably not be posfixing a . to the end of a code...

Reply:
makes sense

Reply:
fixed